src/Network/GRPC/MQTT/Message/Packet.hs

```diff
@@ -174,22 +198,48 @@ wireParsePacket = Packet <$> wireParsePayloadField <*> wireParseMetadataField
           recField = $(reifyRecordField ''Packet 'metadata)
        in Decode.msgOneField recField decodeMessageField
 
--- Packet - Network -------------------------------------------------------------
+-- Packet - Packet Readers ----------------------------------------------------
 
 -- | TODO
 --
 -- @since 0.1.0.0
-packetReader :: TChan LByteString -> ExceptT ParseError IO LByteString
-packetReader = runPacketReader loop
+makePacketReader ::
```

Comment:
I'm a little confused about the `make` nomenclature. Especially given that we have a `PackerReader` data type, intuitively to me, I would expect a function called `makePacketReader` to return a `PacketReader`.

Is there some reason you chose this over something like `readPacket`?

Reply:
Here `TChan LByteString -> WireDecodeOptions -> ExceptT WireDecodeError IO ByteString` is the `PacketReader` even though it is not named. The make prefix is to denote that you are constructing an thing/object that captures a computation that reads all packets into a `ByteString` and is consistent with the analogous functions for other message types like stream chunks.